src/main/java/com/codeup/adlister/controllers/EditProfileServlet.java

```diff
@@ -58,21 +57,6 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         response.sendRedirect("/profile");
 
 
-//        String password = request.getParameter("password");
```

Comment:
In cases like this it's better to remove the code rather than commenting it out. You can always get back this block of code from git in case it's needed.